pkg/api/status.go

```diff
@@ -25,6 +25,7 @@ const (
 	ErrStatusValidatingMerkleRoots  StatusCode = 469
 	ErrStatusFrozenPolicy           StatusCode = 471
 	ErrStatusFrozenConsensus        StatusCode = 472
+	ErrStatusCumulativeFees         StatusCode = 473
 )
 
 func NewErrorFields(status StatusCode, extraInfo string) *ErrorFields {
```

Comment:
I think you need to add this error here too e.g.

```
	case ErrStatusCumulativeFees:
		errFields.Detail = "Cumulative fees are too low"
		errFields.Title = "Cumulative fee too low"
		errFields.Type = arcDocServerErrorsUrl + strconv.Itoa(int(ErrStatusCumulativeFees))
```

Otherwise [here](https://github.com/bitcoin-sv/arc/blob/9a353e6079bd7ebd25f4e48e9fd20a24fea48056/pkg/api/handler/default.go#L532) the error will be a generic error